packages/module/src/catalog/Toolbar/QuickStartCatalogFilterItems.tsx

```diff
@@ -72,7 +72,7 @@ interface QuickStartCatalogFilterSearchWrapperProps {
   onSearchInputChange: any;
 }
 export const QuickStartCatalogFilterSearchWrapper: React.FC<QuickStartCatalogFilterSearchWrapperProps> = ({
-  onSearchInputChange = () => {},
+  onSearchInputChange = (val?:string) => {},
```

Comment:
Can't you create a function in open shift to handle the val being passed in like this?  It takes a parameter of any.

Reply:
@radekkaluzik @dlabaj That's correct, this change isn't needed, it just changes the type of this implementation, but this is then ignored in the function itself.

If you want apply the right type you need to update the type definition of `QuickStartCatalogFilterSearchWrapperProps` above.